vignettes/articles/crane.Rmd

```diff
@@ -0,0 +1,72 @@
+
+#### Where should I add new functions?
+
+- If the function might be useful to R programmers outside of the pharmaceutical industry, add it to **{gtsummary}**.
+- If the function is used specifically to adhere to pharmaceutical standards and/or to create FDA tables, TLG catalog tables, etc., add it to **{crane}**.
+
+## {crane} Functionality
+
+To keep the templates in the {tlg.templates} package simple, if a template cannot be easily constructed with the functions already available in {gtsummary} and {crane} (e.g. many functions or complex modifications are needed) consider adding a function to {crane} to assist.
+
+Currently, functions available in the {crane} package include:
+
+- `tbl_demographics()`, a wrapper for `gtsummary::tbl_summary()` which assists in creating standard DMT01 tables.
+- `tbl_survfit_quantiles()`, which uses `cardx::ard_survival_survfit()` and `gtsummary::tbl_ard_summary()` to create a survival time table (similar to the output of `tern::surv_time()`), useful for creating tables LBT11, LBT11_BL, DORT01, and TTET01 from the TLG catalog.
+
+## Example: Building a demographics table using {gtsummary} vs. {crane}
+
+In the following example, we will compare two simple demographics tables summarizing the `age` and `grade` variables across treatment variable `trt` in the `trial` dataset to demonstrate the differences between {gtsummary} and {crane} functions. The first table will be built using `gtsummary::tbl_summary()` and the second with it's corresponding wrapper function in {crane}, `tbl_demographics()`,
+
+First, we generate a basic demographics table using `tbl_summary()`:
+
+```{r}
+library(gtsummary)
+
+trial |>
+  tbl_summary(
+    by = trt,
+    include = c(age, grade)
+  )
+```
+
+Next the same table is generated using the {crane} function `tbl_demographics()`:
+
+```{r}
+library(crane)
+
+trial |>
+  tbl_demographics(
+    by = trt,
+    include = c(age, grade)
+  )
+```
+
+We can see that the default `tbl_demographics()` table is much more comprehensive than the default `tbl_summary()` output. 
+
+Some of the most noticeable differences in the second table are:
+
+- The Roche gtsummary theme is applied to create a more compact and customized appearance. 
+- More appropriate statistics for a demographics table are displayed - `n` for all variables, and numeric statistics are displayed on multiple rows for the numeric `age` variable, with more included. 
+- Footnotes have been removed
+- Percentages are reported with an extra decimal place. 
+
+To achieve the same result using only {gtsummary} functions many more lines of code would be required, which is why we prefer to use {crane} functions where possible when creating pharmaceutical tables and adding table templates within the {tlg.templates} package.
```

Comment:
Let's not mention tlg.template because crane is external and tlg.templates is not

Reply:
Updated